gui/tasks/migrations/0014_ini_comment_field.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tasks', '0013_init_tasks_timeout'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='initshutdown',
+            name='ini_comment',
+            field=models.TextField(
```

Comment:
Not sure we want a textfield here, it seems to make sense to restrict the size of that field? Thoughts?

Reply:
I did think of that initially, limiting to 100 chars perhaps as in tunables but then thought there might be some users who type longer comments ? But if you think that is better, updating the PR with that.

Reply:
Yes, a limit makes sense to me, it doesnt have to be as low as 100, could be 200

Reply:
Perfect, i'll do that